### ISSUE TITLE
Potential issue with persistant powerups when using Quake3e + missionpack(a)

### ISSUE THREAD

**Issue by KuehnhammerTobias** [ec-/Quake3e#18]
Because it seems to me as if it is currently not a goal of Quake3e running with an updated missionpack mod (am I right?) this issue report is just a reminder if someone wants to use Quake3e for building the baseq3a source with 'ifdef missionpack 'defined.
The issue:
This commit https://github.com/ec-/baseq3a/commit/62f995db7c23e9762ce2c26f61f58bd4ee1b905b#diff-5955bc51f4db0ef76bd33f4d4381022f adds optimized client side prediction but missionpack items aren't handled here, this results in weird behaviour if someone wants to build missionpack game upon baseq3a code. In detail the persistant powerup 'Guard' is not considered to be predicted. Picking up the 'Guard' powerup and the 'Regeneration' powerup will result in unpredicted events (playing a pain sound if health > 100 for example).
Test scenario:
Compile missionpack game mod from baseq3a source. Start missionpack game mod an load map 'mpteam5' (gametype > GT_TEAM). Pick up the 'Guard' powerup and also the 'Regeneration'. While carrying both a pain sound will be played sometimes, although the health is increasing not decreasing.
Since Quake3e/baseq3a are based upon 1.32b people are considering that missionpack should work as well.
Solution:
So eventually we should predict the 'new' missionpack items 'Guard', 'Ammoregen' (?) here too by adding the specific items to cg_predict 'CG_CheckTimers' (https://github.com/ec-/baseq3a/commit/62f995db7c23e9762ce2c26f61f58bd4ee1b905b#diff-5955bc51f4db0ef76bd33f4d4381022fR609).
Well, I know this issue is a bit 'hypothetically', feel free to close this issue if missionpack won't be supported at all.

EDIT: Sorry if this is the wrong repository for this issue, is this issue more appropriate for baseq3a?

Thanks

**Comment by ec-** [comment]
Yes, this is baseq3a issue which is *mostly* focused on VQ3 rather than TA, and I have no plans to support TA in near future (which does NOT mean that I will not support it at all).
In general - this is a good bugreport, no need to duplicate/close it. 
Thank you

**Comment by KuehnhammerTobias** [comment]
I close this issue if it is okay to you, because it is misleading. This issue is also present in baseq3a when picking up the 'Regeneration' powerup. It seems this is a bug in 'CG_CheckTimers' in cg_predict.
At first I did only notice this issue while I attempted to enable compiling the code with MISSIONPACK enabled.
Now I found out it's not related to the Team Arena powerups only.
The issue is not really easy to reproduce, that is why I only noticed it now that baseq3 is affected too. Even recording a demo of the issue doesn't help, because when replaying the demo, the issue doesn't show, as it seems. The only way to experience this bug is by excessively playing maps with the regeneration powerup.
